R/req-options.R

```diff
@@ -49,10 +49,10 @@ req_user_agent <- function(req, string = NULL) {
   check_request(req)
 
   if (is.null(string)) {
-    versions <- c(
-      httr2 = utils::packageVersion("httr2"),
-      `r-curl` = utils::packageVersion("curl"),
-      libcurl = curl::curl_version()$version
+    versions <- list(
```

Comment:
```suggestion
    versions <- c(
```

vapply() was also doing list->vector, if we have three character strings c() works again